### PR TITLE
PSA-40[BNMS] Add Liquibase support for schemas - needed for ENT

### PR DESCRIPTION
**Pull request by aelahi23**
The test I did with the Corda ENT was check that liquibase created a sql output from the xml files, and then I made a modification to the xml script to update the table by adding a new column, I did this to check if the change log is working and if it would record the new script execution and store the new change, which it did so it would allow me to revert to the older database version.

I did the same test on the open source version by copying the database manager tool over to the open source cord-app and repeating the same steps for the ENT version
I also opened the output SQL file to make sure the contents matched the schemas

### CODE REVIEW

File: bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-createTable.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<changeSet author="R3 Corda" id="Create the table">
+    <createTable tableName="Schema">
```
Comment (ivanr3) [inline bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-createTable.xml]: table name should be `membership_states`

File: bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-createTable.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<changeSet author="R3 Corda" id="Create the table">
+    <createTable tableName="Schema">
+        <column name="member" type="NVARCHAR(255)"/>
+        <column name="BNO" type="NVARCHAR(255)"/>
+        <column name="status" type="NVARCHAR(255)"/>
```
Comment (ivanr3) [inline bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-createTable.xml]: please make sure that name of the columns match the actual schema: https://github.com/corda/corda-solutions/blob/master/bn-apps/memberships-management/membership-service-contracts-and-states/src/main/kotlin/com/r3/businessnetworks/membership/states/Schemas.kt

File: bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-PendingMembershipRequestSchemaV1.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<changeSet author="R3 Corda" id=" create PersistentPendingMembershipRequest table">
+    <createTable tableName="pending_membership_requests">
+        <column name="id" type="INT">
```
Comment (ivanr3) [inline bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-PendingMembershipRequestSchemaV1.xml]: This value needs to be autogenerated. Something along these lines: https://www.liquibase.org/documentation/changes/add_auto_increment.html

File: bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-Include.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<databaseChangeLog>
+    <include file="Schema.changelog-createTable.xml"/>
+    <include file="Schema.changelog-PendingMembershipRequestSchemaV1.xml"/>
```
Comment (ivanr3) [inline bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-Include.xml]: Also can you please rename `Schema.changelog-createTable.xml` to be consistent with the other one?

Comment (aelahi23) [reply bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-Include.xml]: I've made it consistent now

**Pull request merged**